package advanced.alfa.lesson10;

import java.io.IOException;

public class Task31 {
    public static void main(String[] args) throws IOException {
        //work1
        //new advanced.alfa.lesson10.work1.Runner().run();

        //work2
        new advanced.alfa.lesson10.work2.Runner().run();

        //work3
        //new advanced.alfa.lesson10.work3.Runner().run();
    }
}
